staging/knative/Chart.yaml

```diff
@@ -1,19 +1,14 @@
 apiVersion: v2
 name: knative
-version: 0.6.0
+version: 1.0.1
 description: "Kubernetes-based platform to build, deploy, and manage modern serverless workloads"
 home: https://knative.dev/
 maintainers:
 - name: akirillov
+- name: adityajariwala
 icon: https://raw.githubusercontent.com/knative/docs/master/docs/images/logo/rgb/knative-logo-rgb.jpg
-appVersion: "v0.22.3"
+appVersion: "v1.9.2"
 dependencies:
-  - name: eventing
```

Comment:
just making sure it was intentional to remove eventing and eventing-sources charts. would this be a breaking change since they were being deployed by default before? would it impact users? (for upgrades)

Reply:
@gracedo This [cm.yaml file in Kommander-applications](https://github.com/mesosphere/kommander-applications/blob/main/services/knative/0.6.0/defaults/cm.yaml) overrides the values.yaml to have eventing and eventing-sources set to false, I don't believe they are ever deployed or used. We were on a relatively recent version of serving, but eventing and eventing-sources were not really ever upgraded. Based on those reasons I just chose to remove them.

Reply:
Do we need to worry about the case where a user overrode the values to enable either of those? (no idea if this was ever done)